### PR TITLE
addon:netlify-workload

### PR DESCRIPTION
**Pull request by oeular**
Signed-off-by: Xueliang Yang <yangxueliang@cmbchina.com>

### Checklist

I have:

- [x] Title of the PR starts with type (e.g. `[Addon]` or `[Trait]`).
- [x] Updated/Added any relevant [documentation] and [examples].
- [ ] Unit/E2E Tests added.

### CODE REVIEW

File: addons/netlify/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+version: v10.5.1
+description: Interact with Netlify from the comfort of your CLI.
+url: https://github.com/netlify/cli
+icon: https://github.com/netlify/cli/blob/main/cli.png
+tags:
+  - extended_workload
+
+deployTo:
+  control_plane: true
+  runtime_cluster: true
```
Comment (wonderflow) [inline addons/netlify/metadata.yaml:11]: No need for runtime cluster

File: addons/netlify/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+metadata:
+  name: netlify-app
+  namespace: default
+spec:
+  components:
+    - name: netlify-comp
+      type: netlify
+      properties:
+         sitename: deploy-from-vela
+         token: <Here needs your own token from https://app.netlify.com/user/applications/personal>
```
Comment (wonderflow) [inline addons/netlify/README.md:23]: Please use a secret for token

File: addons/netlify/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: netlify
+version: v10.5.1
+description: Interact with Netlify from the comfort of your CLI.
+url: https://github.com/netlify/cli
+icon: https://github.com/netlify/cli/blob/main/cli.png
```
Comment (wangyikewxgm) [inline addons/netlify/metadata.yaml:5]: Please use `raw.githubcontent` prefix URL for icon. For this is: https://raw.githubusercontent.com/netlify/cli/main/cli.png

File: addons/netlify/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+name: netlify
+version: v10.5.1
```
Comment (wangyikewxgm) [inline addons/netlify/metadata.yaml:2]:
```suggestion
version: 10.5.1
```

File: addons/netlify/definitions/netlify.cue [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            backofflimit: *5 | int
+            completions:  *1 | int
+            // +usage=Specify the website name on app.netlify.com
+            sitname: *"deploy-from-vela" | string
+           // +usage=Specify the token got from app.netlify.com
+            token: *"your-own-token" | string
+           // +usage=Specify the direcotry name for saving the resources from git or other repo supporting git-cli
+            reponame: *"oeular.github.io" | string
+           // +usage=Specify the repo address your resources of the frontend service  store
+            gitrepo: *"https://github.com/oeular/oeular.github.io.git" | string
```
Comment (wangyikewxgm) [inline addons/netlify/definitions/netlify.cue:60]: No need set set the default value?

**Review by wangyikewxgm — COMMENTED** [review]
Please write a doc for this verified addon or move it to experiment addon.

File: addons/netlify/definitions/netlify.cue (hunk 5 above)
Comment (oeular) [reply addons/netlify/definitions/netlify.cue:60]: It may be a good example for user.

**Comment by oeular** [comment]
@wangyikewxgm  There is a verified demo in README.md. Is it needed to write other details? Give me some details.

**Comment by wangyikewxgm** [comment]
> @wangyikewxgm There is a verified demo in README.md. Is it needed to write other details? Give me some details.

Have test this addon manually, I think this pr is ready to merge. Please write a doc for this verified addon,  like this:https://kubevela.net/docs/reference/addons/ai

**Review by wangyikewxgm — APPROVED** [review]

**Pull request merged**